src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -522,11 +526,6 @@ def _set_gradient_checkpointing(self, module, value=False):
             - 0 for tokens that are **masked**.
 
             [What are attention masks?](../glossary#attention-mask)
-        position_ids (`torch.LongTensor` of shape `(batch_size, sequence_length)`, *optional*):
-            Indices of positions of each input sequence tokens in the position embeddings. Selected in the range `[0,
-            config.max_position_embeddings - 1]`.
-
-            [What are position IDs?](../glossary#position-ids)
```

Comment:
We can't just remove an argument of a model in a release, that's too breaking.

Reply:
Is it safe to assume we can remove the documentation as it doesn't make sense (config.max_position_embeddings is not defined for example), and keep `position_ids=None` in the function signature to prevent breaking change. Just to be sure the functions that we can break are model functions, typically: `BloomModel.forward`, but it's okay to break `BloomBlock.forward`?

Reply:
Also do we have a nice helper to deprecate? Something like a decorator where we can define arguments that are deprecated, with a warning message? all the examples I've seen uses:
```
if position_ids is not None:
    warnings.warn(
            "Use of `position_ids` is deprecated, will be removed in v5.0.0",
            FutureWarning,
        )
```

Reply:
No, that's the way to do it.